DevCycle/SSEConnection.swift

```diff
@@ -56,3 +56,22 @@ class Handler: EventHandler {
         Log.error("Streaming connection had an error: " + error.localizedDescription)
     }
 }
+
+public struct SSEMessage {
+    struct Data {
+        var etag: String?
+        var lastModified: Int?
+        var type: String?
+    }
+
+    var data: Data
+
+    init?(from dictionary: [String: Any]) throws {
```

Comment:
does this actually throw?

Reply:
it's a relic from the past. I had this method throw when either `etag`, `lastModified` weren't in the dictionary but I changed that and forgot to remove `throws`. I'll clean it pup

Reply:
ok so I made it such that it doesn't throw on init but `SSEMessage` can be nil. This way people won't have to wrap the instantiation in a `do` `catch`

Reply:
okay instead of having an failable init, I'm just making it throw. It's easier to test this way and give more info when stuff goes wrong